components/admin/Header.tsx

```
@@ -8,31 +8,34 @@ interface HeaderProps {
         />
       </div>
-      <div className="flex items-center space-x-2">
-    
-        <button className="p-2 rounded-full hover:bg-gray-100 dark:hover:bg-gray-700 transition-colors duration-200">
-          <Bell className="h-6 w-6 text-gray-500 dark:text-gray-400" />
+
+      <div className="flex items-center space-x-1 sm:space-x-2 md:space-x-3">
+        <button
+          className="p-1.5 sm:p-2 md:p-2.5 rounded-full hover:bg-gray-100 dark:hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2 dark:focus:ring-offset-gray-800 transition-all duration-200"
+          aria-label="Notifications"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)